src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb

```diff
@@ -0,0 +1,98 @@
+                                ImmutableArray.Create({
+                                    New TaggedText("Keyword", "void"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Class", "Console"),
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Method", "WriteLine"),
+                                    New TaggedText("Punctuation", "("),
+                                    New TaggedText("Keyword", "string"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Parameter", "value"),
+                                    New TaggedText("Punctuation", ")"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Punctuation", "("),
+                                    New TaggedText("Punctuation", "+"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Text", "18"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Text", "overloads"),
+                                    New TaggedText("Punctuation", ")")})),
+                            QuickInfoSection.Create("DocumentationComments",
+                                ImmutableArray.Create({New TaggedText("Text", "Writes the specified string value, followed by the current line terminator, to the standard output stream.")})),
+                            QuickInfoSection.Create("Exception",
+                                ImmutableArray.Create({
+                                    New TaggedText("Text", "Exceptions"),
+                                    New TaggedText("LineBreak", "\r\n"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Namespace", "System"),
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Namespace", "IO"),
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Class", "IOException")}))}))
+
+            Dim trackingSpan = New Mock(Of ITrackingSpan) With {
+                .DefaultValue = DefaultValue.Mock
+            }
+
+            Dim intellisenseQuickInfo = IntellisenseQuickInfoBuilder.BuildItem(trackingSpan.Object, codeAnalysisQuickInfoItem)
+
+            Assert.NotNull(intellisenseQuickInfo)
+
+            Dim container = Assert.IsType(Of Adornments.ContainerElement)(intellisenseQuickInfo.Item)
+            Assert.Equal(3, container.Elements.Count())
+            Assert.Equal(ContainerElementStyle.Stacked, container.Style)
+
+            Assert.Collection(container.Elements,
+                    New Action(Of Object)() {
```

Comment:
Is the new Action()() thing required? I would have imagined VB would figure it all out.

Reply:
I get the following error if I do not specify New Action(Of Object)()
BC36645	Data type(s) of the type parameter(s) in method 'Public Shared Overloads Sub Collection(Of T)(collection As IEnumerable(Of T), ParamArray elementInspectors As Action(Of T)())' cannot be inferred from these arguments. Specifying the data type(s) explicitly might correct this error.